pkg/standalone/standalone.go

```diff
@@ -1013,3 +1049,37 @@ func parseVersionFile(fromDir string) (string, string) {
 
 	return versions[daprRuntimeFilePrefix], versions[dashboardFilePrefix]
 }
+
+func getDefaultRegistry() string {
+	imageRegistryName := githubContainerRegistryName
+	if val := os.Getenv("DEFAULT_IMAGE_REGISTRY_MODE"); val != "" {
+		switch strings.ToLower(val) {
+		case githubContainerRegistryName:
+			imageRegistryName = githubContainerRegistryName
+		case dockerContainerRegistryName:
+			imageRegistryName = dockerContainerRegistryName
+		default:
+			print.FailureStatusEvent(os.Stdout,
+				fmt.Sprintf("Environment variable %s can only be set to %s and %s", "DEFAULT_IMAGE_REGISTRY_MODE", "GHCR", "DOCKERHUB"))
+			os.Exit(1)
```

Comment:
return error ... 
check if error where it is called and propagate the error upstream ...

Reply:
Caller should be in a position to take a call on termination, instead of we taking a call here.